cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
You need to register the Movie model with the Django admin site to manage it through the admin interface. Add the following line: `admin.site.register(Movie)` after importing the Movie model.